src/website/src/app/documentation/demos/datepicker/demos/datepicker-api.demo.ts

```diff
@@ -1,34 +1,40 @@
 /*
- * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Component } from '@angular/core';
 
 const HTML_EXAMPLE = `
-<form clrForm clrLayout="vertical">
-  <clr-date-container>
-    <input type="date" clrDate name="demo" [(ngModel)]="demo">
-  </clr-date-container>
+<form clrForm>
+    <clr-date-container>
+        <label>Basic Demo</label>
+        <input type="date" clrDate name="demo" [(ngModel)]="demo">
+    </clr-date-container>
 </form>
 `;
 
 const MIN_EXAMPLE = `
-<form clrForm clrLayout="vertical">
+<form clrForm>
   <clr-date-container>
-    <input type="date" clrDate name="demo" [(ngModel)]="demo" min="2019-11-11">
+    <label>Min date: 2019-11-17</label>
+    <input type="date" clrDate name="demo" [(ngModel)]="demo" min="2019-11-17">
   </clr-date-container>
 </form>
 `;
 const MAX_EXAMPLE = `
+<form clrForm>
   <clr-date-container>
-    <input type="date" clrDate name="demo" [(ngModel)]="demo" max="2019-11-11">
+    <label>Max date: 2019-11-19</label>
+    <input type="date" clrDate name="demo" [(ngModel)]="demo" max="2019-11-19">
   </clr-date-container>
 </form>
 `;
 const MIN_MAX_EXAMPLE = `
+<form clrForm>
   <clr-date-container>
-    <input type="date" clrDate name="demo" [(ngModel)]="demo" min="2019-11-11" max="2019-12-12">
+    <label>Min date: 2019-11-17 AND Max date: 2019-11-19</label>
```

Comment:
Last nit: when you squash copy these labels into the actual demos above so they are the same.